sysutils/clamav-server/Portfile

```diff
@@ -1,10 +1,11 @@
-# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: t; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:sw=4:ts=4:sts=4
```

Comment:
This still seems to be incorrect. The original was correct.

Reply:
Okay, I copied and pasted from https://guide.macports.org/#development.practices.